### ISSUE TITLE
Missing license

### ISSUE THREAD

**Issue by olleharstedt** [AmiBlitz/AmiBlitz3#4]
Can we perhaps add a license to this repository? Personally I'd prefer MIT, but I don't know if there is an already existing license.

**Comment by honitos** [comment]
Good point. As I remember, Bernd Roesch got the permission to reverse engineer the code. But as the former coder Simon Armstrong released all his old blitz2 related stuff (https://github.com/nitrologic) it seems to me, we could use a opensource license for this "fork".
But one should ask the other contributors to AB2/AB3 first.

**Comment by khval** [comment]
I the official site says “Apache License V2.0”
https://sourceforge.net/projects/amiblitz3/

**Comment by vazhnov** [comment]
Guys, could you please create the `LICENSE` file? It is possible to generate it with "Add file" button, which is located near green "<> Code" button on the "main" page https://github.com/AmiBlitz/AmiBlitz3
